docs/devops-guide/manual.md

```diff
@@ -190,11 +206,13 @@ EOF
 
 Start the videobridge with:
 ```sh
+unzip target/jitsi-videobridge-2.1-SNAPSHOT-archive.zip
```

Comment:
This `2.1-SNAPSHOT` should be changed to `{version}`, to make it independent from specific releases.
The same applies below.